models/dsn_helpers.go

```diff
@@ -40,30 +40,61 @@ func FindDSNByServiceIDandPMMAgentID(q *reform.Querier, serviceID, pmmAgentID, d
 		return "", err
 	}
 
-	var agentType AgentType
+	var agentTypes []AgentType
 	switch svc.ServiceType {
 	case MySQLServiceType:
-		agentType = MySQLdExporterType
-	case MongoDBServiceType:
-		agentType = MongoDBExporterType
+		agentTypes = append(
+			agentTypes,
+			QANMySQLSlowlogAgentType,
+			QANMySQLPerfSchemaAgentType,
+			MySQLdExporterType,
+			QANPostgreSQLPgStatementsAgentType,
```

Comment:
let's remove all agents not related to MySQL, The same with PostgreSQL and MongoDB